R/data.R

```diff
@@ -0,0 +1,16 @@
+#' Sector level carbon budgets for scenarios
```

Comment:
I am not familiar with this way of documenting data and attaching them to the package. Is this the way to keep them internal?

Reply:
No, I believe this is general data documentation. It will return some basic information if you run e.g. `?scenario_thresholds`. I can try to remove it.

Reply:
basically functionally the same as the file `R/alignment_scores_values.R` which was already in the repo. But collecting that type of information for all data sets...